product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -93,15 +117,19 @@ Now that your endpoint service is created, you can connect it to the cluster VPC
 
 1. Under **Service category**, select **Other endpoint services**.
 
-1. Under **Service Name**, enter the name of the endpoint service that you created earlier (`com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc`). To verify whether you successfully allowed access to the endpoint, select **Verify service**.
+1. Under **Service Name**, enter the name of the endpoint service that you created earlier:
+   
+   -  If following the example using your own AWS account: (`com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc` i - If using BigAnimal's cloud: the service name provided in your BigAnimal cluster's details). 
```

Comment:
Please double check this edit. The text was a bit garbled but I think I got it right.